man/meson.build

```diff
@@ -168,73 +191,69 @@ foreach tuple : xsltproc.found() ? [['systemd.directives', '7', systemd_directiv
                 command : xslt_cmd + [custom_html_xsl, '@INPUT@'],
                 depends : [custom_entities_ent, p2],
                 install : want_html and have_lxml,
-                install_dir : docdir / 'html')
+                install_dir : docdir / 'html',
+        )
         html_pages += p3
 endforeach
 
 # Cannot use run_target because those targets are used in depends
 # Also see https://github.com/mesonbuild/meson/issues/368.
-man = custom_target(
-        'man',
-        output : 'man',
-        depends : man_pages,
-        command : [echo])
-
-html = custom_target(
-        'html',
-        output : 'html',
-        depends : html_pages,
-        command : [echo])
+man = custom_target('man', output : 'man', depends : man_pages, command : [echo])
+
+html = custom_target('html', output : 'html', depends : html_pages, command : [echo])
 
 if rsync.found()
         run_target(
                 'doc-sync',
                 depends : man_pages + html_pages,
-                command : [rsync, '-rlv',
-                           '--delete-excluded',
-                           '--include=man',
-                           '--include=*.html',
-                           '--exclude=*',
-                           '--omit-dir-times',
-                           meson.current_build_dir(),
-                           get_option('www-target')])
+                command : [
+                        rsync,
+                        '-rlv',
+                        '--delete-excluded',
+                        '--include=man',
+                        '--include=*.html',
+                        '--exclude=*',
+                        '--omit-dir-times', meson.current_build_dir(),
+                        get_option('www-target'),
+                ],
```

Comment:
Meh.

Reply:
@annacrombie This looks wrong. The cmdline option formatting should keep args isolated if they already are (unless they end with `=`).